drivers/can/Kconfig.sam

```diff
@@ -8,3 +8,15 @@ config CAN_SAM
 	bool "Atmel SAM CAN driver"
 	default $(dt_compat_enabled,$(DT_COMPAT_ATMEL_SAM_CAN))
 	select CAN_MCAN
+
+if CAN_SAM
+
+config CAN_MAX_FILTER
+	int "Maximum number of concurrent active filters"
+	default 5
+	range 1 32
+	help
+	  Defines the array size of the callback/msgq pointers.
+	  Must be at least the size of concurrent reads.
+
+endif # CAN_SAM
```

Comment:
This seems to be a separated commit or could even be on a different PR.
Could you clarify?